CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
-## TBD
-* Updated `hotels-oss-parent` version to 4.2.0 (was 4.0.0). Contains updates to the copyright header.
+## [15.0.0] - TBD
+
+### Updated
```

Comment:
`Updated` is not a valid tag as per https://keepachangelog.com/en/1.0.0/.

This should go in `Changed`.